src/main/features/settlement-agreement/views/settlement-agreement-confirmation.njk

```diff
@@ -0,0 +1,37 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+{% from "externalLink.njk" import externalLink %}
+
+{% extends "layout.njk" %}
+
+{% block content %}
+<div class="grid-row">
+  <div class="column-two-thirds">
+
```

Comment:
lots of empty lines for no reason. Can we delete a lot of them to make the code more compact?